nfdi4ing/profiles/README.md

```diff
@@ -0,0 +1,25 @@
+# /nfdi4ing/profiles
+This [W3ID](https://w3id.org) provides a persistent URI namespace for metadata profiles created with the NFDI4Ing Metadata Profile Service via the AIMS platform. More information on the project AIMS can be found at <https://www.aims-projekt.de>.
+
+## Uses
+This namespace is related to <https://w3id.org/nfdi4ing>.
+
+## Contact
+General contact: [contact@nfdi4ing.de](mailto:contact@nfdi4ing.de)
+
+This space is administered by:  
+
+**Marc Fuhrmans**  
+Research Data Services   
+Information technology, Research and Development   
+Technical University of Darmstadt   
+[University and State Library Darmstadt](https://ulb.tu-darmstadt.de)   
+Darmstadt, Germany   
+ORCID: [0000-0002-9826-018X](https://orcid.org/0000-0002-9826-018X)
+
+**TBA**  
+XXXXXX
+xxx-xxx
+xxx-xxx-xxx
+City
```

Comment:
who should be added here?

Reply:
@Marius: please also include your ORCID (currently line 25).

Reply:
I updated the contact details.

Reply:
@mfuhrmans, @Zack-83 approved by me. thanks to @MattGroenewald, contact details look good.